### PR TITLE
Standalone pyinstaller interpreter for openmotics source code

### PR DESCRIPTION
**Pull request by ptuytsch**
Pyinstaller starts up and import included python source code files and runs this code instead of running the "included" code of pyinstaller. This makes it updateable.

### CODE REVIEW

File: pyinstaller/package/clean.sh [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+
+rm -rf gw_service.zip*
```
Comment (LnL7) [inline pyinstaller/package/clean.sh:4]: Any reason not to keep this in the makefile?

File: pyinstaller/create-package.sh [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+logger "Starting the openmotics gateway package"
+logger "\$0: $0"
+logger "app path: $APP_PATH"
+logger "app data path: $APP_DATA_PATH"
+logger "app tmp path: $APP_TMP_PATH"
+logger "config file: $MAIN_CONFIG_FILE"
+
+# -- start gateway service --
+killall -9 "openmotics_service"
```
[the remark this answers is not in the document]
Comment (ptuytsch) [reply pyinstaller/create-package.sh:40]: yes correct!

File: pyinstaller/create-package.sh [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+# setting up the package folder
+mkdir -p package
+cp -r dist package
+cd package
+mv dist bin
+
+# adding the package info file
+echo '{
+    "name": "APP_openmotics_gateway",
+    "version": [0, 0, 2],
```
[the remark this answers is not in the document]
Comment (ptuytsch) [reply pyinstaller/create-package.sh:13]: This version is used by RensonCore to know what the version is of  the package. And to be honest, I would not use the version of the gateway code that is installed, I would keep this separate from the gateway version. The reason being, that the gateway will be able to update itself in the future to a newer version. Then this version will not be in sync with the actual installed version. (There is also no realistic way to update this version to a newer one without explicitly installing a new version through RensonCore) Furthermore, this version could indicate which python packages are installed within the pyinstaller build.

[the remark this answers is not in the document]
Comment (ptuytsch) [reply pyinstaller/create-package.sh:13]: But indeed, There is need for a better solution to define the version of the package :)

**Pull request closed without merging**